### ISSUE TITLE
Build Error 

### ISSUE THREAD

**Issue by ap4472** [mrousavy/react-native-mmkv#585]
I updated my compileSDK from 31 to 33 in Android Studio. Due to this change I needed to upgrade my react-native version. After that while I am trying to build my application I had these 2 errors. My react-native-mmkv version is 2.10.2 and my react native version is 0.71.3.

1: Task failed with an exception.
-----------
* Where:
Build file '/node_modules/react-native-mmkv/android/build.gradle' line: 49

* What went wrong:
A problem occurred evaluating project ':react-native-mmkv'.
> Could not set unknown property 'namespace' for extension 'android' of type com.android.build.gradle.LibraryExtension.

==============================================================================

2: Task failed with an exception.
-----------
* What went wrong:
A problem occurred configuring project ':react-native-mmkv'.
> compileSdkVersion is not specified. Please add it to build.gradle

**Comment by theafolayan** [comment]
You should use expo build properties config plugin. 


``` 
"plugins": [
             "expo-build-properties",
        {
          "android": {
            "compileSdkVersion": 33,
            "targetSdkVersion": 33,
            "buildToolsVersion": "33.0.0"
          }
        }
     
```

**Comment by gtdrag** [comment]
I'm getting the same error. Were you able to resolve it?

**Comment by aybarska** [comment]
i am not using expo. Any solution?

**Comment by mrousavy** [comment]
Fixed in latest MMKV 2.12
